sigopt/cli/commands/experiment/create.py

```diff
@@ -0,0 +1,29 @@
+from sigopt.logging import print_logger
+
+from ...arguments import experiment_file_option
+from ...utils import create_experiment_from_validated_data
+from .base import experiment_command
+
+
+def print_help(experiment):
+  msg = f'''
+You can now start workers for this experiment with the following CLI command:
+> sigopt start-worker {experiment.id}
+
+Or use the python client library:
+
+  #/usr/bin/env python3
+  import sigopt
+  experiment = sigopt.get_experiment({experiment.id!r})
+  for run in experiment.loop():
+    with run:
+      ...
+'''
+  print_logger.info(msg)
+
+@experiment_command.command()
+@experiment_file_option
+def create(experiment_file):
+  '''Create a SigOpt Experiment.'''
+  experiment = create_experiment_from_validated_data(experiment_file)
+  print_help(experiment)
```

Comment:
is this always printing the help? or is this like a magic invocation if the user adds `--help`?

Reply:
It does print help every time (how to start the workers for the experiment). See this doc for expected usage https://docs.google.com/document/d/1V9unQDk6XsGqOnYyCEp8x54jI63yLNPAwRspsNE1LJA/edit#

The CLI is not currently being designed with automation in mind since the python client currently has all of the features necessary to do so. It would be a reasonable undertaking to support automation via the CLI but it hasn't been scoped.

Reply:
I'll take that into consideration